packages/enhanced/src/lib/container/ContainerEntryModule.ts

```diff
@@ -262,12 +285,16 @@ class ContainerEntryModule extends Module {
       ])};`,
       `var init = ${runtimeTemplate.basicFunction('shareScope, initScope', [
         `if (!${RuntimeGlobals.shareScopeMap}) return;`,
-        `var name = ${JSON.stringify(this._shareScope)}`,
-        `var oldScope = ${RuntimeGlobals.shareScopeMap}[name];`,
-        `if(oldScope && oldScope !== shareScope) throw new Error("Container initialization failed as it has already been initialized with a different share scope");`,
-        `${RuntimeGlobals.shareScopeMap}[name] = shareScope;`,
-        `return ${RuntimeGlobals.initializeSharing}(name, initScope);`,
-      ])};`,
+				`if (!${federationGlobal}) return;`,
+				`var name = ${JSON.stringify(this._shareScope)}`,
+				//`var oldScope = ${RuntimeGlobals.shareScopeMap}[name];`,
+				// no need to verify shareScope, because shareScope will be proxied by global shareScope
+				//`if(oldScope && oldScope !== shareScope) throw new Error("Container initialization failed as it has already been initialized with a different share scope");`,
```

Comment:
@2heal1 what happens if initialization is called again, since it cannot / will no throw error?

Reply:
at past , the reason why it throw error while repeat init  is that shareScopeMap is passed by init parameters .  

But now ,we can get shareScope from global , so it may no need to add this validate . 

But if there are other reasons why not allow to repeat initialization, we can add throwing error again.